development/docker/CentOS7Slim.Dockerfile

```diff
@@ -0,0 +1,73 @@
+# Arguments allowed to be used in FROM have to come
+# before the first stage
+ARG CPPCHECK_VERSION=2.5
+
+# Import cppcheck. COPY --from cannot used variables.
+# Define a local name
+FROM neszt/cppcheck-docker:${CPPCHECK_VERSION} AS upstream_cppcheck
+
+# Base
```

Comment:
It just occurred to me that it's not really obvious why we have picked CentOS.
Could you add a comment that we did this to match with the platform used by [conda-forge](https://github.com/chrisburr/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml#L142)?